fuzzing/reversion/reversion_statistics.go

```diff
@@ -0,0 +1,188 @@
+package reversion
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/crytic/medusa/chain"
+	"github.com/crytic/medusa/compilation/abiutils"
+	"github.com/crytic/medusa/fuzzing/config"
+	fuzzerTypes "github.com/crytic/medusa/fuzzing/contracts"
+	"github.com/crytic/medusa/logging"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"os"
+	"path/filepath"
+)
+
+type ReversionMeasurer struct {
+	incomingReportsQueue chan *RevertReport
```

Comment:
```suggestion
	incomingReportsQueue chan *RevertReport
```
I would call this something like TxCallMetrics (it may not have reverted) and change RevertArtifact to RevertReport as artifact is often confusing to other devs IME

Reply:
me gusta